misk/web/tabs/loader/src/sagas/loaderSaga.tsx

```diff
@@ -13,6 +13,17 @@ import { all, call, put, takeEvery, takeLatest } from "redux-saga/effects"
 import {
   dispatchLoader, IAction, IActionType, LOADER
 } from "../actions"
+import { IMultibinder } from "../utils/binder"
+
+function * handleCacheTabEntries (action: IAction<IActionType, { MiskBinder: IMultibinder}>) {
+  const { MiskBinder } = action.payload
+    yield put(dispatchLoader.success({ adminTabComponents: { blah: "blah"} , staleTabCache: false }))
```

Comment:
what’s a tab cache?

Reply:
Currently keeping a cache of each tab's code in the React/Redux state. Was part of testing a hypothesis that we could render a component given their compiled code.